testing/workflows/components/workflows.libsonnet

```diff
@@ -142,9 +149,14 @@
                   "/usr/local/bin/checkout.sh",
                 ],
                 args: [
-                  srcDir,
+                  srcRootDir,
                 ],
-                env: prow_env,
+                env: prow_env + [{
+                  "name": "EXTRA_REPOS",
+                  // TODO(jlewi): Once kubeflow/testing#12 is submitted pin
+                  // to kubeflow/testing HEAD
+                  "value": "tensorflow/k8s@HEAD;kubeflow/testing@HEAD:12",
```

Comment:
Do you need kubeflow/testing as EXTRA_REPOS? I think in https://github.com/kubeflow/testing/pull/12/files#diff-c385e0edb32af89cbebb9b1abbe2024cR91, it always checks out that repo?

Reply:
Yes. Because we need a PR that isn't checked in yet. Also as noted in the TODO for kubeflow/testing#12 I think it makes sense to stop giving special treatement to kubeflow/testing.